low-level/sensors/impl/centos_7/disk_monitor.py

```diff
@@ -1100,7 +1122,11 @@ def _get_drive_fault_info(self, path):
             cmd = f"sudo smartctl -d scsi -A {self._drive_by_device_name[path]}"
         else:
             cmd = f"sudo smartctl -A {self._drive_by_device_name[path]}"
-        response, _, _ = self._run_command(cmd)
+        response, err, retcode = self._run_command(cmd)
+        if retcode != 0 and err:
+            self._iem.iem_fault("SMARTMONTOOL_ERROR")
+        if retcode == 0:
+            self._iem.iem_fault_resolved("SMARTMONTOOL_ERROR", "SMARTMONTOOL_AVAILABLE")

```

Comment:
same comment as udisks (line 228). and same for hdparm too below

Reply:
Done.